website/package.json

```
@@ -22,7 +22,7 @@
     "@docusaurus/preset-classic": "2.4.1",
     "@foxglove/eslint-plugin": "1.0.1",
     "@foxglove/rostime": "1.1.2",
-    "@foxglove/schemas": "1.6.2",
+    "@foxglove/schemas": "github:foxglove/schemas#4cd9d40f61a46ec6a209e0c0f5d85db3cc9e6488",
```

After https://github.com/foxglove/schemas/pull/159 has been submitted and before merging this PR, update the version here and `yarn install` to update the `yarn.lock`.

```suggestion
    "@foxglove/schemas": "1.6.7",
```